test/recipes/70-test_sslversions.t

```diff
@@ -153,8 +152,8 @@ sub modify_supported_versions_filter
             } elsif ($testtype == WITH_TLS1_4) {
                     $ext = pack "C5",
                         0x04, # Length
-                        #TODO(TLS1.3): Fix before release
-                        0x7f, 0x1c; #TLSv1.3 (draft 28)
+                        0x03, 0x05, #TLSv1.4
+                        0x03, 0x04; #TLSv1.3
```

Comment:
We're going from one version listed to two versions listed (but the length is still 4 bytes)?

Reply:
This change was actually separated out into a separate commit in the PR because it is also a bug fix. The length was incorrect before (it should have been 2), and it wasn't listing TLSv1.4 which it should have done. This fixes it. The length of 4, which was wrong before, is now right.